### PR TITLE
Fix #190: Support for Google Places values in venue-related methods and types

### LINKED ISSUE #190: Support for Google Places values in venue-related methods and types
Added the fields `google_place_id` and `google_place_type` to the classes [Venue](https://core.telegram.org/bots/api#venue), [InlineQueryResultVenue](https://core.telegram.org/bots/api#inlinequeryresultvenue), [InputVenueMessageContent](https://core.telegram.org/bots/api#inputvenuemessagecontent) and the optional parameters `google_place_id` and `google_place_type` to the method [sendVenue](https://core.telegram.org/bots/api#sendvenue) to support Google Places as a venue API provider.

### PR DESCRIPTION
**Pull request by madhead**

### CODE REVIEW

File: tgbotapi.core/src/commonMain/kotlin/dev/inmo/tgbotapi/requests/send/SendVenue.kt [hunk 1]
```diff
@@ -25,7 +25,9 @@ data class SendVenue(
     @SerialName(addressField)
     val address: String,
     @SerialName(foursquareIdField)
-    val foursquareId: String? = null,
+    val foursquareId: FoursquareId? = null,
+    @SerialName(googlePlaceIdField)
+    val googlePlaceId: GooglePlaceId? = null,
```
Comment (InsanusMokrassar) [inline tgbotapi.core/src/commonMain/kotlin/dev/inmo/tgbotapi/requests/send/SendVenue.kt:30]: Here must be `googlePlaceType` too (https://core.telegram.org/bots/api#sendvenue)

File: tgbotapi.extensions.api/src/commonMain/kotlin/dev/inmo/tgbotapi/extensions/api/send/SendVenue.kt [hunk 2]
```diff
@@ -15,13 +15,14 @@ suspend fun TelegramBot.sendVenue(
     longitude: Double,
     title: String,
     address: String,
-    foursquareId: String? = null,
+    foursquareId: FoursquareId? = null,
+    googlePlaceId: GooglePlaceId? = null,
```
Comment (InsanusMokrassar) [inline tgbotapi.extensions.api/src/commonMain/kotlin/dev/inmo/tgbotapi/extensions/api/send/SendVenue.kt:19]: The same comment as for `SendVenue`

File: tgbotapi.extensions.utils/src/commonMain/kotlin/dev/inmo/tgbotapi/extensions/utils/extensions/venue/Foursquare.kt [hunk 3]
```diff
@@ -16,7 +16,7 @@ fun Venue(
     title: String,
     address: String,
     foursquare: Foursquare
```
Comment (InsanusMokrassar) [inline tgbotapi.extensions.utils/src/commonMain/kotlin/dev/inmo/tgbotapi/extensions/utils/extensions/venue/Foursquare.kt:18]: I am not pretty sure that it is impossible to use `google_place_id` and `google_place_type` with foursquare. So, I think we must add these parameters too

**Review by InsanusMokrassar — CHANGES_REQUESTED** [review]

File: tgbotapi.extensions.utils/src/commonMain/kotlin/dev/inmo/tgbotapi/extensions/utils/extensions/venue/Google.kt [hunk 4]
```diff
@@ -0,0 +1,27 @@
+package dev.inmo.tgbotapi.extensions.utils.extensions.venue
+
+import dev.inmo.tgbotapi.types.*
+import dev.inmo.tgbotapi.types.location.StaticLocation
+import dev.inmo.tgbotapi.types.venue.Venue
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+val Venue.googlePlace: GooglePlace?
```
Comment (InsanusMokrassar) [inline tgbotapi.extensions.utils/src/commonMain/kotlin/dev/inmo/tgbotapi/extensions/utils/extensions/venue/Google.kt:9]: :+1:

File: tgbotapi.extensions.utils/src/commonMain/kotlin/dev/inmo/tgbotapi/extensions/utils/extensions/venue/Google.kt [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+fun Venue(
+    location: StaticLocation,
+    title: String,
+    address: String,
+    googlePlace: GooglePlace
+) = Venue(location, title, address, googlePlaceId = googlePlace.id, googlePlaceType = googlePlace.type)
+
+@Serializable
+data class GooglePlace(
```
Comment (InsanusMokrassar) [inline tgbotapi.extensions.utils/src/commonMain/kotlin/dev/inmo/tgbotapi/extensions/utils/extensions/venue/Google.kt:22]: :+1:

File: tgbotapi.core/src/commonMain/kotlin/dev/inmo/tgbotapi/requests/send/SendVenue.kt (hunk 1 above)
Comment (madhead) [reply tgbotapi.core/src/commonMain/kotlin/dev/inmo/tgbotapi/requests/send/SendVenue.kt:30]: Why it's `googlePlaceType`, but `foursquareId`?

File: tgbotapi.extensions.utils/src/commonMain/kotlin/dev/inmo/tgbotapi/extensions/utils/extensions/venue/Foursquare.kt (hunk 3 above)
Comment (madhead) [reply tgbotapi.extensions.utils/src/commonMain/kotlin/dev/inmo/tgbotapi/extensions/utils/extensions/venue/Foursquare.kt:18]: But there is `Google.kt` for that

File: tgbotapi.core/src/commonMain/kotlin/dev/inmo/tgbotapi/requests/send/SendVenue.kt (hunk 1 above)
Comment (InsanusMokrassar) [reply tgbotapi.core/src/commonMain/kotlin/dev/inmo/tgbotapi/requests/send/SendVenue.kt:30]: Screenshot_20201106-020518_Vivaldi-01
Yes, you are right, according to docs we must have both fields `googlePlaceType` and `fourSquareType` in this request

Comment (madhead) [reply tgbotapi.core/src/commonMain/kotlin/dev/inmo/tgbotapi/requests/send/SendVenue.kt:30]: Should I also add Foursquare type?

Comment (InsanusMokrassar) [reply tgbotapi.core/src/commonMain/kotlin/dev/inmo/tgbotapi/requests/send/SendVenue.kt:30]: I think so

**Pull request merged**